### PR TITLE
Fix Clippy and RERUN_MEMORY_LIMIT env variable default

### PR DESCRIPTION
**Pull request by haixuanTao**
Quick Fix of `dora-rerun` that is not parsing `RERUN_MEMORY_LIMIT` in the right way and fixing a clippy error

### CODE REVIEW

File: tool_nodes/dora-rerun/src/main.rs [hunk 1]
```diff
@@ -21,87 +21,94 @@ fn main() -> Result<()> {
     let mut options = SpawnOptions::default();
 
     let memory_limit = std::env::var("RERUN_MEMORY_LIMIT")
-        .context("Could not read image height")?
+        .context("Could not read image height")
+        .unwrap_or("25%".into())
```
Comment (phil-opp) [inline tool_nodes/dora-rerun/src/main.rs:25]: I think we should handle `VarError::NotPresent` differently than `VarError::NotUnicode`. In the former case, it's fine to provide a default, but the latter should result in an error IMO.

File: tool_nodes/dora-rerun/src/main.rs [hunk 2]
```diff
@@ -21,87 +21,94 @@ fn main() -> Result<()> {
     let mut options = SpawnOptions::default();
 
     let memory_limit = std::env::var("RERUN_MEMORY_LIMIT")
-        .context("Could not read image height")?
+        .context("Could not read image height")
```
Comment (phil-opp) [inline tool_nodes/dora-rerun/src/main.rs:24]: The context message doesn't really make sense here (image height?).

**Review by phil-opp — APPROVED** [review]

File: tool_nodes/dora-rerun/src/main.rs (hunk 1 above)
Comment (haixuanTao) [reply tool_nodes/dora-rerun/src/main.rs:25]: That makes sense. I'll do the same for the rest of the env variables.

File: tool_nodes/dora-rerun/src/main.rs (hunk 2 above)
Comment (haixuanTao) [reply tool_nodes/dora-rerun/src/main.rs:24]: Indeed sorry!

**Review by phil-opp — APPROVED** [review]

**Pull request merged**